packages/framework/src/Resources/views/Admin/Form/orderItems.html.twig

```diff
@@ -2,82 +2,84 @@
-                                <th class="text-right"><span class="table-main__column-with-info">{{ 'Unit price including VAT'|trans }} ({{ currencySymbolByCurrencyId(order.currency.id) }})</span></th>
-                                <th class="text-right">{{ 'Amount'|trans }}</th>
-                                <th class="text-right">{{ 'Unit'|trans }}</th>
-                                <th class="text-right">{{ 'VAT rate (%)'|trans }}</th>
-                                <th class="text-center">
-                                    <span class="display-inline-block" style="width: 80px">
-                                        {{ 'Set prices manually'|trans }}
-                                        <i class="svg svg-info cursor-help js-tooltip"
-                                           data-toggle="tooltip" data-placement="bottom"
-                                           title="{{ 'All prices have to be handled manually if checked, otherwise the unit price without VAT and the total prices for that item will be recalculated automatically.'|trans }}"
-                                        ></i>
-                                    </span>
-                                </th>
-                                <th class="text-right">{{ 'Unit price excluding VAT'|trans }} ({{ currencySymbolByCurrencyId(order.currency.id) }})</th>
-                                <th class="text-right">{{ 'Total including VAT'|trans }} ({{ currencySymbolByCurrencyId(order.currency.id) }})</th>
-                                <th class="text-right">{{ 'Total excluding VAT'|trans }} ({{ currencySymbolByCurrencyId(order.currency.id) }})</th>
-                                <th></th>
-                        </tr>
-                        </thead>
-                        <tbody
-                            id="js-order-items"
-                            data-prototype="{{ orderItemMacro.orderItem(form.itemsWithoutTransportAndPayment.vars.prototype, null, null)|escape }}"
-                            data-order-product-add-url="{{ url('admin_order_addproduct', { orderId: order.id }) }}"
-                        >
-                            {% for productItem in order.productItems %}
-                                {{ orderItemMacro.orderItem(form.itemsWithoutTransportAndPayment[productItem.id], productItem.id, productItem) }}
-                            {% endfor %}
+        <div class="form-group">
+            <div class="form-group__title">{{ 'Order items'|trans }}</div>
+            <div class="form-line">
 
-                            {% for orderItemId, orderItemForm in form.itemsWithoutTransportAndPayment %}
-                                {% if not orderItemForm.rendered %}
-                                    {{ orderItemMacro.orderItem(orderItemForm, orderItemId, null) }}
-                                {% endif %}
-                            {% endfor %}
+                    <div class="table-touch-always">
```

Comment:
I think that this indentation is broken. Is'n it?

Reply:
Yes, there was one end div missing too. Fixed.